graylog2-web-interface/src/views/pages/ExtendedSearchPage.jsx

```diff
@@ -154,14 +155,22 @@ const ExtendedSearchPage = ({ route, location = { query: {} }, router, searchRef
         .push(SearchActions.refresh.listen(refreshIfNotUndeclared))
         .push(ViewActions.search.completed.listen(refreshIfNotUndeclared));
       return null;
-    });
+    }).catch((error) => setErrors([error]));
```

Comment:
I wrote the `SearchExectuionErrors` component in a way it can handle all types of errors. initially I've tried to add the `.catch()` to `SearchActions.execute(executionState)` but I prefer catching and informing the user about all possible errors (which are not part of a 403 / 404 request).

I've also tried to add the `.catch()` before the `.finally()`. but this results in some test failures I couldn't fix easily.